src/SFA.DAS.EmployerIncentives.Commands/CollectionCalendar/ActivateCollectionPeriodCommandHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.CollectionPeriod;
+using SFA.DAS.EmployerIncentives.Commands.Services;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Commands.CollectionCalendar
+{
+    public class ActivateCollectionPeriodCommandHandler : ICommandHandler<ActivateCollectionPeriodCommand>
+    {
+        private readonly ICollectionCalendarService _collectionCalendarService;
+        public ActivateCollectionPeriodCommandHandler(ICollectionCalendarService collectionCalendarService)
+        {
+            _collectionCalendarService = collectionCalendarService;
+        }
+
+        public async Task Handle(ActivateCollectionPeriodCommand command, CancellationToken cancellationToken = default)
+        {
+            var collectionCalendar = await _collectionCalendarService.Get();
+
+            collectionCalendar.ActivatePeriod(command.CollectionPeriodYear, command.CollectionPeriodNumber, command.Active);
```

Comment:
Pass in a CollectionPeriod value object.  Note the existing CollectionPeriod  has too many properties on it's constructor.  there is a tech debt item for that.   So for now create a new constructor with just periodNumber , and year.

collectionCalendar.ActivatePeriod could be just collectionCalendar.SetActive(CollectionPeriod  period)

and add a condition to see if command.Activeis true

i.e.

if(command.Active)
{
 collectionCalendar.SetActive(CollectionPeriod  period)
}

// else just ignore since there is no use case for updating anything other than active CollectionPeriods yet.